syntax = "proto3";

message LocationMessage {
    string person_id = 2;
    string longitude = 3;
    string latitude = 4;
    string creation_time = 5;
}

message LocationById {
    string location_id = 1;
}

message LocationResponse {
    string id = 1;
    string person_id = 2;
    string longitude = 3;
    string latitude = 4;
    string creation_time = 5;
}

service LocationService {
    rpc GetById(LocationById) returns (LocationResponse) {}
    rpc Create(LocationMessage) returns (LocationMessage) {}
}